### DISCUSSION TITLE
Is ChangeNotifier at risk in any way?

### DISCUSSION THREAD

**Discussion by mark8044** [rrousselGit/riverpod#1263] [Q&A] [answered]
I literally just now read this comment https://github.com/rrousselGit/river_pod/issues/1023#issuecomment-1024895682

My implementation is littered with ChangeNotifier implementation and its quite frankly amazing and Im super stocked with it.  

But after reading that comment it begs the question, is there any risk that it would be depreciated in the future?  

I don't want to invest so heavily in something that might go away one day.

Please say NO, it will stay forever 👍 :)

**Comment by rrousselGit** [comment, marked as answer]
There's no reason to remove it.
